content/20.treatments.md

```diff
@@ -136,19 +136,46 @@ Summarize the state of the antiviral approach.
 
 #### Protease Inhibitors
 
-Why it may be useful
+The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.
 
-##### Protease Inhibitor 1
+##### N3
+N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.
 
 ###### Anticipated Mechanism
+N3 inhibits Mpro through binding to its substrate pocket.
 
 ###### Current Evidence
 
-A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
+N3 was first designed computationally to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases. 
```

Comment:
Is there a good reference for this statement?

Reply:
I just added two references in the last commit.